scripts/build_native_deps.sh

```diff
@@ -309,9 +317,15 @@ build_guile()
         $MAKE install
         # Build shared libraries for srfi modules.
         cd "$GUILE_INSTALL/lib"
-        for srfi in 1-v-3 4-v-3 13-14-v-3 60-v-2; do
+        SRFI_MODULES=(
+            1-v-3     # List library.
+            4-v-3     # Homogeneous numeric vector datatypes.
+            13-14-v-3 # String & character set library.
+            60-v-2    # Integers as bits.
+        )
```

Comment:
Interesting. Not sure if it's worth documenting here, but now that you typed it...